api/controllers/user.js

```diff
@@ -0,0 +1,29 @@
+const db = require("../models/index");
+
+const Users = db.users;
+const { hashPassword, isPasswordValid, generateToken } = require("../helpers/authHelper");
+
+exports.signup = async (req, res) => {
+  try {
+    // get values from body
+    const {
+      username, email, password, confirmPassword
+    } = req.body;
+    // checks if password and confirmPasword matche
+    if (password !== confirmPassword) {
+      return res.status(403).json({ response: "confirm password not matched with password" });
+    }
+
+    // hash password
+    const hash = await hashPassword(confirmPassword);
+    const token = await generateToken({ email });
+    console.log(token);
```

Comment:
You're not suppose to add console.log() to the code na

Reply:
How did that escaped eslint and codacy check... Will fix now

Reply:
Okay

Reply:
it has been fixed ..this is flagged outdated ..it must be one of the previous commits that have been fixed  in the new commits